lib/ecs_deploy/instance_fluctuation_manager.rb

```diff
@@ -6,6 +6,8 @@ module EcsDeploy
   class InstanceFluctuationManager
     attr_reader :logger
 
+    MAX_DESCRIBABLE_CONTAINER_COUNT = 10
```

Comment:
[nits] `MAX_UPDATABLE_CONTAINER_INSTANCE_COUNT`
I think the word "container" is usually associated with a docker container in the context of ECS.